tests/tests_geomstats/test_pre_shape.py

```diff
@@ -766,3 +766,9 @@ def test_parallel_transport(self):
         is_horizontal = space.is_horizontal(transported, end_point)
         self.assertTrue(gs.all(is_tangent))
         self.assertTrue(gs.all(is_horizontal))
+
+        transported = metric.parallel_transport(
+            tan_a[0], point, end_point=end_point[0], n_steps=150, step="rk4"
```

Comment:
could we check that both versions of the parallel transport give the same result?

Reply:
What do you mean both versions? Do mean ladder methods? This check is done for the hypersphere.

Reply:
Sorry I wasn't clear. i mean a version with initial_point and end_point, and a version with initial_point and initial_tangent_vec.